packages/ada-language-server/mdx/package.yaml

```
@@ -0,0 +1,17 @@
+name: mdx-analyzer
+description: MDX language support
+homepage: https://github.com/mdx-js/mdx-analyzer/tree/main/packages/language-server
+licenses: MIT
+languages: MDX
+categories:
+  - LSP
+
+source:
+  id: pkg:npm/@mdx-js/language-server@0.1.0
```

The `@` needs to [percent-encoded](https://en.wikipedia.org/wiki/URL_encoding), otherwise everything after it becomes the version specifier:

```suggestion
  id: pkg:npm/%40mdx-js/language-server@0.1.0
```